unittests/SourceKit/SwiftLang/CursorInfoTest.cpp

```diff
@@ -395,3 +408,51 @@ TEST_F(CursorInfoTest, DISABLED_CursorInfoMustWaitDueTokenRace) {
   EXPECT_EQ(FooOffs, Info.Offset);
   EXPECT_EQ(strlen("fog"), Info.Length);
 }
+
+TEST_F(CursorInfoTest, CursorInfoCancelsPreviousRequest) {
+  // TODO: This test case relies on rdar://80582770 that the first cursor info
+  // request takes longer to execute than it takes time to schedule the second
+  // request. If that radar is fixed, we need to find a new way to cause slow
+  // type checking.
+  const char *SlowDocName = "slow.swift";
```

Comment:
It's fine to list the radar number, but we should rephrase this comment so that it doesn't sound like you need access to the radar to understand the issue.  Suggestion:

> TODO: This test case relies on the following snippet being slow to type check so that the first cursor info request takes longer to execute than it takes time to schedule the second request. If that is fixed, we need to find a new way to cause slow type checking. rdar://80582770

Reply:
I thought the comment was understandable even without radar access when I wrote it. But I was probably in my own mental bubble. Thanks for the suggestion.